data/src/main/java/com/teamzzong/hacker/data/model/response/auth/ResponsePhotoUrl.kt

```diff
@@ -0,0 +1,13 @@
+package com.teamzzong.hacker.data.model.response.auth
+
+import com.teamzzong.hacker.domain.entity.auth.PhotoUrl
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponsePhotoUrl(
+    val profileImage: String
+) {
+    fun toPhotoUrl() : PhotoUrl {
+        return PhotoUrl(profileImage)
+    }
```

Comment:
```suggestion
fun toPhotoUrl() = PhotoUrl(profileImage)
```

이정도만 해도 충분히 컴파일러 단에서 타입추론할 수 있을 것 같아. 이건 형이 의도해서 적은 거일수도 있을 것 같은데 괜히 남기는 거일수도? 형이 적은 방식대로 하면 다음에 유지보수할 때 리턴 패러미터 형식이 한 눈에 잘보여서 수정하기에는 더 편하다는 장점이 있어, 단점은 줄이 좀 길어진다는거? 

앞으로 이런 매핑함수 같은 경우에는 어떻게 적어야할까 하는 생각도 있어서 의견 서로 나눠보는 것도 좋을 것 같음

Reply:
토의 완료